homeassistant/components/scene/deconz.py

```diff
@@ -0,0 +1,53 @@
+"""
+Support for deCONZ scenes.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/scene.deconz/
```

Comment:
URL is wrong.

Reply:
This one is correct?

Reply:
Yes, you're correct, it's correct! Sorry! :blush: